### PR TITLE
Add auto labeler for hubble-cli

### PR DESCRIPTION
**Pull request by aanm**
This will allow us to generate a dedicated CHANGELOG for any PR that
affects hubble-cli.

### CODE REVIEW

**Review by michi-covalent — APPROVED** [review]
fascinating

File: .github/workflows/auto-labeler.yaml [hunk 1]
```diff
@@ -74,3 +74,13 @@ jobs:
...
             })
+
+  auto-labeler:
+    name: Auto Labeler
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - id: label-the-PR
```
Comment (brlbil) [inline .github/workflows/auto-labeler.yaml]: Step missing name, validation is complaining about it https://github.com/cilium/cilium/actions/runs/8940546996/job/24558885412?pr=32343

**Review by brlbil — CHANGES_REQUESTED** [review]

**Review by brlbil — APPROVED** [review]

**Review by chancez — APPROVED** [review]

**Review by joestringer — APPROVED** [review]

**Comment by joestringer** [comment]
```
The configuration file (path: .github/labeler.yml) was not found locally, fetching via the api
Warning: The config file was not found at .github/labeler.yml. Make sure it exists and that this action has the correct access rights.
Error: HttpError: Not Found
Error: Not Found
```

`labeler.yml`, not `labeler.yaml`. PR #32406 should fix it.

**Pull request merged**